.github/workflows/main.yml

```
@@ -91,6 +91,7 @@ jobs:
           - {ROS_DISTRO: melodic, TARGET_WORKSPACE: 'industrial_ci/mockups/test_clang_tidy', CLANG_TIDY: pedantic, EXPECT_EXIT_CODE: 1}
           - {ROS_DISTRO: melodic, TARGET_WORKSPACE: 'industrial_ci/mockups/test_clang_tidy', CLANG_TIDY: pedantic, CLANG_TIDY_ARGS: "-checks=-*,modernize-use-nullptr", EXPECT_EXIT_CODE: 1}
 
+          - {ROS_DISTRO: noetic, PYLINT_ARG: "--errors-only", PYLINT_EXCLUDE: "sample_talker.py", PYLINT_CHECK: "true"}
```

```suggestion
          - {ROS_DISTRO: noetic, PYLINT_ARGS: "--errors-only", PYLINT_EXCLUDE: "sample_talker.py", PYLINT_CHECK: "true"}
```

if you look closely, you see that `--errors-only` was not part of the command:
```
  $ ( source /root/target_ws/install/setup.bash && cd /root/target_ws && pylint /root/target_ws/src/industrial_ci/industrial_ci/setup.py /root/target_ws/src/industrial_ci/industrial_ci/python/industrial_ci/__init__.py /root/target_ws/src/industrial_ci/industrial_ci/python/industrial_ci/travis.py; )
```

---

:face_palm: Ahh, thank you for catching!

Wish ba/sh could return something like undefined variable (but that's contradictory to the feature I guess)..